src/Sarif.Multitool.Library/Rules/RuleResources.Designer.cs

```diff
@@ -60,6 +60,96 @@ internal class RuleResources {
+        /// </summary>
+        internal static string ADO1003_ProvideDriver_Note_Default_Text {
+            get {
+                return ResourceManager.GetString("ADO1003_ProvideDriver_Note_Default_Text", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to {0}: This &apos;driver&apos; object does not provide a &apos;fullName&apos; property. This property is required by the {1} service..
+        /// </summary>
+        internal static string ADO1003_ProvideFullName_Note_Default_Text {
+            get {
+                return ResourceManager.GetString("ADO1003_ProvideFullName_Note_Default_Text", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to {0}: This &apos;automationDetails&apos; object does not provide an &apos;id&apos; property. This property is required by the {1} service..
+        /// </summary>
+        internal static string ADO2010_ProvideAutomationDetailsId_Note_Default_Text {
+            get {
+                return ResourceManager.GetString("ADO2010_ProvideAutomationDetailsId_Note_Default_Text", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to {0}: This &apos;sarifLog&apos; object&apos;s &apos;runs&apos; array contains {1} elements, which exceeds the limit of {2} imposed by {3}..
+        /// </summary>
+        internal static string Base1001_MaximumRunsCount_Note_Default_Text {
+            get {
+                return ResourceManager.GetString("Base1001_MaximumRunsCount_Note_Default_Text", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to {0}: This &apos;run&apos; object does not provide a &apos;results&apos; property. This property is required by the {1} service..
+        /// </summary>
+        internal static string Base1002_ProvideResultsArray_Note_Default_Text {
+            get {
+                return ResourceManager.GetString("Base1002_ProvideResultsArray_Note_Default_Text", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to {0}: This &apos;tool&apos; object does not provide a &apos;driver&apos; property. This property is required by the {1} service..
+        /// </summary>
+        internal static string Base1003_ProvideDriver_Note_Default_Text {
+            get {
+                return ResourceManager.GetString("Base1003_ProvideDriver_Note_Default_Text", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to {0}: This &apos;driver&apos; object does not provide a &apos;fullName&apos; property. This property is required by the {1} service..
+        /// </summary>
```

Comment:
Same here: change "a" to "an"

Reply:
fullName doesn't begin with a vowel sound